src/actions/mod.rs

```diff
@@ -447,16 +451,27 @@ impl ActionHandler {
                 return;
             }
         };
-        let config = self.fmt_config.lock().unwrap();
+
+        let mut config = self.fmt_config.lock().unwrap().get_rustfmt_config().clone();
+        let file_lines = format!(r#"[{{
+            "file": "{}",
+            "range": [{}, {}]
+        }}]"#, path.to_str().unwrap(), range.start.line + 1, range.end.line + 1);
+        config.set().file_lines(file_lines.parse().unwrap());
```

Comment:
I guess we need a new API for rustfmt to take an optional range over a text input, rather than using this file/range approach